File: carrito_v4/src-gen/windows/Model/ShoppingCart.cs
//Start of user code imports
using System;
using System.Runtime.Serialization;
using System.Collections.Generic;
//End of user code

namespace CarritoDeCompras.Model
{
    [DataContract]
    public class ShoppingCart 
    {
		//The Id property is marked as the Primary Key
        [SQLite.PrimaryKey, SQLite.AutoIncrement]
        [DataMember]
        public int Id { get; set; }

		[DataMember]
		public int idCart { get; set; }

		[DataMember]
		public string price { get; set; }

		[DataMember]
		public int quantity { get; set; }

		[DataMember]
		public int idProduct { get; set; }

		[DataMember]
		public double syncTime { get; set; }

		/**
		* Empty Constructor.
		*/
		public ShoppingCart (){
		}
	
		/**
		* Constructor.
		*/
		public ShoppingCart (int idCart, string price, int quantity, int idProduct, double syncTime){
			this.idCart = idCart;     
			this.price = price;     
			this.quantity = quantity;     
			this.idProduct = idProduct;     
			this.syncTime = syncTime;     
		}

		public String getTotalCarPrice() {
		// Start of user code getTotalCarPrice
		// TODO should be implemented
		// End of user code
			return;
		}

		public int getCartProductCount() {
		// Start of user code getCartProductCount
		// TODO should be implemented
		// End of user code
			return;
		}
    }
}

File: carrito_v4/src-gen/android/app/src/main/java/com/example/carritodecompras/beans/ShoppingCart.java

package com.example.carritodecompras.beans;

//Start of user code imports
import java.io.Serializable;
import java.math.BigDecimal;
//End of user code

public class ShoppingCart  implements Serializable{
	

	/**
	* the idCart attribute.
	*/
	private Integer idCart;


	/**
	* the price attribute.
	*/
	private String price;


	/**
	* the quantity attribute.
	*/
	private Integer quantity;


	/**
	* the idProduct attribute.
	*/
	private Integer idProduct;


	/**
	* the syncTime attribute.
	*/
	private BigDecimal syncTime;
	

	/**
	* Empty Constructor.
	*/
	public ShoppingCart (){
	}

	/**
	* Constructor.
	*/
	public ShoppingCart (Integer idCart, String price, Integer quantity, Integer idProduct, BigDecimal syncTime){
		this.idCart = idCart;
		this.price = price;
		this.quantity = quantity;
		this.idProduct = idProduct;
		this.syncTime = syncTime;
	}

	public Integer getIdCart() {
       return this.idCart;
     }

	public String getPrice() {
       return this.price;
     }

	public Integer getQuantity() {
       return this.quantity;
     }

	public Integer getIdProduct() {
       return this.idProduct;
     }

	public BigDecimal getSyncTime() {
       return this.syncTime;
     }
 
     public void setIdCart(Integer idCart) {
        this.idCart = idCart;
     }

     public void setPrice(String price) {
        this.price = price;
     }

     public void setQuantity(Integer quantity) {
        this.quantity = quantity;
     }

     public void setIdProduct(Integer idProduct) {
        this.idProduct = idProduct;
     }

     public void setSyncTime(BigDecimal syncTime) {
        this.syncTime = syncTime;
     }

	public String getTotalCarPrice() {
	// Start of user code getTotalCarPrice
	// TODO should be implemented
	// End of user code
		return;
	}

	public int getCartProductCount() {
	// Start of user code getCartProductCount
	// TODO should be implemented
	// End of user code
		return;
	}

}
